test/Microsoft.Azure.WebJobs.Host.FunctionalTests/JobHostTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Azure.WebJobs.Host.FunctionalTests.TestDoubles;
+using Microsoft.Azure.WebJobs.Host.Storage;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.FunctionalTests
+{
+    public class JobHostTests
+    {
+        // Test that we can do static initialization before runtime initialization. 
+        [Fact]
+        public async Task StaticInit()
```

Comment:
This new SDK test demonstrates how I expect Script to leverage this work. It's verifying all this churn actually accomplishes something practical :)